GAE/src/org/waterforpeople/mapping/app/web/SurveyAssemblyServlet.java

```diff
@@ -198,7 +198,7 @@ private boolean assembleFormWithJackson(Long formId) {
             log.debug("Finishing assembly of " + formId);
             form.setStatus(Survey.Status.PUBLISHED);
             if (form.getWebForm()){
-                boolean webForm = WebForm.validWebForm(surveyGroupDao.getByKey(form.getObjectId()), form, questionDao.listQuestionsBySurvey(form.getObjectId()));
+                boolean webForm = WebForm.validWebForm(surveyGroupDao.getByKey(form.getSurveyGroupId()), form, questionDao.listQuestionsBySurvey(form.getObjectId()));
```

Comment:
should we also change the second use of `form.getObjectId() ` ?

Reply:
nope that one is ok. 👍